src/main/java/org/folio/rest/impl/FiscalYearsApi.java

```diff
@@ -56,6 +61,10 @@ public void postFinanceFiscalYears(String lang, FiscalYear fiscalYear, Map<Strin
       .exceptionally(fail -> handleErrorResponse(handler, fail));
   }
 
+  private boolean isFiscalYearValid(FiscalYear fiscalYear) {
```

Comment:
We need the same valiation in PUT method

Reply:
fixed